src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -4,29 +4,32 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.util.Properties;
+import java.util.Scanner;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service";
+    private static String USERNAME;
+    private static String PASSWORD;
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
 
     static {
         try {
             Class.forName(JDBC_DRIVER);
         } catch (ClassNotFoundException e) {
             throw new RuntimeException("Can't find SQL Driver", e);
         }
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your mySQL login:");
+        USERNAME = scanner.nextLine();
+
+        System.out.println("Enter your mySQL password:");
+        PASSWORD = scanner.nextLine();
     }
 
-    public static Connection getConnection() {
+    public static Connection getConnection() throws SQLException {
```

Comment:
better leave as it was, so we can have information specific to bad connection, not just general can't do something